Unable to access node view

**Description**

When clicking on a node to access the node details, an error is raised and the view is empty.

Error raised in the javascript console:

```javascript
app.e447a296.js:118 Error: [$resource:badcfg] http://errors.angularjs.org/1.5.8/$resource/badcfg?p0=get&p1=object&p2=array&p3=GET&p4=api%2Fdocker%2F2%2Fnodes
    at Error (native)
    at http://localhost:9000/js/app.e447a296.js:6:412
    at http://localhost:9000/js/app.e447a296.js:375:50
    at http://localhost:9000/js/app.e447a296.js:131:20
    at m.$eval (http://localhost:9000/js/app.e447a296.js:145:347)
    at m.$digest (http://localhost:9000/js/app.e447a296.js:142:420)
    at m.$apply (http://localhost:9000/js/app.e447a296.js:146:113)
    at l (http://localhost:9000/js/app.e447a296.js:97:322)
    at J (http://localhost:9000/js/app.e447a296.js:102:34)
    at XMLHttpRequest.t.onload (http://localhost:9000/js/app.e447a296.js:103:4)
```

**Steps to reproduce the issue:**

1. Connect to a swarm mode enabled cluster
2. Go to the swarm view
3. Click on any node name

Any other info e.g. Why do you consider this to be a bug? What did you expect to happen instead?

The node details view should be displayed.

**Technical details:**

* Portainer version: 1.11.4
* Portainer Docker image tag (latest/arm/windows...): latest
* Target Docker version (the host/cluster you manage): 17.03.0-ce
* Platform (windows/linux): Linux

--- a/app/components/swarm/swarm.html
+++ b/app/components/swarm/swarm.html
@@ -215,7 +215,7 @@
           </thead>
           <tbody>
             <tr dir-paginate="node in (state.filteredNodes = (nodes | filter:state.filter | orderBy:sortType:sortReverse | itemsPerPage: state.pagination_count))">
-              <td><a ui-sref="node({id: node.ID})">{{ node.Hostname }}</a></td>
+              <td><a ui-sref="node({id: node.Id})">{{ node.Hostname }}</a></td>
               <td>{{ node.Role }}</td>
               <td>{{ node.CPUs / 1000000000 }}</td>
               <td>{{ node.Memory|humansize }}</td>

--- a/app/components/swarm/swarmController.js
+++ b/app/components/swarm/swarmController.js
@@ -75,7 +75,7 @@ function ($scope, Info, Version, Node, Pagination) {
     var node = {};
     node.name = info[offset][0];
     node.ip = info[offset][1];
-    node.id = info[offset + 1][1];
+    node.Id = info[offset + 1][1];
     node.status = info[offset + 2][1];
     node.containers = info[offset + 3][1];
     node.cpu = info[offset + 4][1].split('/')[1];

